doc/userguide/setting-up-ipsinline-for-linux.rst

```diff
@@ -1,7 +1,8 @@
 Setting up IPS/inline for Linux
 ================================
 
-In this guide will be explained how to work with Suricata in layer3 inline mode and how to set iptables for that purpose.
+In this guide will be explained how to work with Suricata in layer3
```

Comment:
None on the output.  Its like HTML, the section determines the formatting.  Its just much nicer to write/read/edit when its properly justified in your editor IMO.  Probably shouldn't have committed this one though as it has no modification.